frontend/src/views/web/challenge/phases.html

```diff
@@ -1,19 +1,27 @@
 <section ng-repeat="item in challenge.phases.results" class="ev-sm-container ev-view challenge-container">
     <div class="ev-md-container ev-card-panel ev-z-depth-5 ev-challenge-view phase-card">
         <div class="row">
-            <div class="col s12">
-                <span class="fs-18 text-med-black"><strong>{{item.name}}</strong></span>
+            <div class="col s11 m11">
+                <div class="col s12">
```

Comment:
Wrap col in row please. This is bad practice for grid implementation.
FYI: Cover col-12 in A ROW. For every new row wrap col in 'row' class. For example
```
<div class="row">
    <div class="col s8">
         <div class="row">
               <div class="col s12"></div>
         </div>
     </div>
    <div class="col sm4"></div>
</div>
```
Hope you get it.